server/movieList.js

```diff
@@ -1,103 +1,105 @@
 
 module.exports = {
-  movies: `The Shawshank Redemption
-  The Godfather
-  The Godfather, Part II
-  The Dark Knight
-  12 Angry Men
-  Schindlers List
-  Pulp Fiction
-  The Lord of the Rings: The Return of the King
-  The Good, the Bad, and the Ugly
-  Fight Club
-  The Lord of the Rings: The Fellowship of the Ring
-  Forrest Gump
-  Star Wars: Episode V: The Empire Strikes Back
-  Inception
-  The Lord of the Rings: The Two Towers
-  One Flew Over The Cuckoos Nest
-  GoodFellas
-  The Matrix
-  The Seven Samurai
-  Star Wars
-  City of God
-  Se7en
-  The Silence of the Lambs
-  Its A Wonderful Life
-  Life is Beautiful
-  The Usual Suspects
-  Leon, aka The Professional
-  Saving Private Ryan
-  Spirited Away
-  Coco
-  American History X
-  Interstellar
-  Once Upon a Time in the West
-  The Green Mile
-  Psycho
-  Casablanca
-  City Lights
-  Intouchables
-  Modern Times
-  The Pianist
-  Raiders of the Lost Ark
-  The Departed
-  Rear Window
-  Terminator 2: Judgment Day
-  Back to the Future
-  Whiplash
-  Gladiator
-  The Lion King
-  The Prestige
-  Memento
-  Apocalypse Now
-  Alien
-  The Great Dictator
-  Sunset Boulevard
-  Cinema Paradiso
-  Dr. Strangelove
-  The Lives of Others
-  Grave of the Fireflies
-  Paths of Glory
-  Django Unchained
-  The Shining
-  WALL-E
-  American Beauty
-  Princess Mononoke
-  The Dark Knight Rises
-  Blade Runner 2049
-  Oldboy
-  Witness For the Prosecution
-  Aliens
-  Once Upon a Time in America
-  Das Boot
-  Dangal
-  Citizen Kane
-  Vertigo
-  North By Northwest
-  Star Wars: Episode VI - Return of the Jedi
-  Braveheart
-  Reservoir Dogs
-  M
-  Requiem for a Dream
-  Your Name
-  Like Stars on Earth
-  Amelie
-  A Clockwork Orange
-  Lawrence of Arabia
-  Amadeus
-  Double Indemnity
-  Eternal Sunshine of the Spotless Mind
-  Taxi Driver
-  To Kill A Mockingbird
-  Full Metal Jacket
-  Singin In The Rain
-  2001: A Space Odyssey
-  Toy Story
-  3 Idiots
-  The Sting
-  Toy Story 3
-  Inglourious Basterds
-  The Bicycle Thief
-  The Kid`,
+  movies:
```

Comment:
I'm curious why this file is just exporting 100 lines of strings?

Reply:
Its used as the grounds for randomly generated movie names for the titles